packages/contentstack-bulk-publish/src/commands/cm/entries/unpublish.js

```diff
@@ -19,6 +19,7 @@ class UnpublishCommand extends Command {
     unpublishFlags.onlyAssets = false;
     unpublishFlags.onlyEntries = true;
     unpublishFlags.apiVersion = unpublishFlags['api-version'] || '3';
+    unpublishFlags.includeVariants = unpublishFlags['include-variants'] || unpublishFlags.includeVariants || false;
```

Comment:
unpublishFlags['include-variants'] || unpublishFlags.includeVariants || false;

Do we really need to all different cases here "unpublishFlags['include-variants'] " this will be enough right, we are setting the default value as "false" in the below flag configuration itself, we can avoid second and third or case here